sdk/keyvault/azure-keyvault-certificates/dev_requirements.txt

```diff
@@ -2,6 +2,7 @@
 -e ../../../tools/azure-devtools
 -e ../../../tools/azure-sdk-tools
 -e ../../identity/azure-identity
+-e ../azure-keyvault-nspkg
```

Comment:
That should not be necessary for dev, since all init will contain pkgutil line, nspkg is really interesting only for Python 2 correct support

Reply:
At least it won't be necessary once the `azure-keyvault-nspkg` package has been published to PyPi :)

Reply:
It was necessary to unblock CI, which was failing to install the dependency (it hasn't been published).